### ISSUE TITLE
QA Ph4: Accessibility_Modal close button is not labeled accurately 

### ISSUE THREAD

**Issue by Remiqa** [GoogleChrome/ioweb2016#912]
With VoiceOver enabled, the live modal close button is not labeled accurately as a close button, it is only labeled as button.  

Steps: 
1. Enable VoiceOver in accessibility settings
2. Go to https://io-webapp-staging.appspot.com/io2016/
3. Expand live modal 
4. Navigate through available options using the tab key 
5. Note that the close button is simply labeled as "button" 

Chrome 50 Mac OS X 10.11 

![gio_912](https://cloud.githubusercontent.com/assets/7585604/15232583/33030eaa-1857-11e6-981a-15fa6fb494da.png)
